jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -271,6 +272,18 @@ public String getRequestURI()
         @Override
         public Object getAttribute(String name)
         {
+            //Servlet Spec 9.4.2 no forward attributes if a named dispatcher
+            if (_named != null && name != null && name.startsWith(__FORWARD_PREFIX))
+                return null;
+
+            //Servlet Spec 9.4.2 must return the values from the original request
+            if (name != null && name.startsWith(__FORWARD_PREFIX))
```

Comment:
I don't think this always works.

If we did a forward -> named forward -> forward, then I think this breaks. 

Then the last forward would see the null from the named dispatch wrapper and assume that it is the first forward dispatch.